spec/helpers/find/goto_preview_helper_spec.rb

```diff
@@ -66,6 +66,66 @@ module Find
       end
     end
 
+    describe '#goto_provider_value' do
+      subject do
+        goto_provider_value(param_form_key:, params:)
+      end
+
+      context 'params is empty' do
+        let(:params) { {} }
+
+        it 'returns falsey' do
+          expect(subject).to be_nil
+        end
+      end
+
+      context 'params has goto_provider set to "true"' do
+        let(:params) { { goto_provider: 'true' } }
+
+        it 'returns truthy' do
```

Comment:
```suggestion
        it 'returns the value "true"' do
```
A bit of rogue copy and pasta